library/aesce.h

```diff
@@ -30,11 +30,18 @@
 
 #include "mbedtls/aes.h"
 
-
+#if !defined(MBEDTLS_HAVE_ARM64)
 #if defined(MBEDTLS_HAVE_ASM) && defined(__GNUC__) && \
```

Comment:
This conflicts with the change to check_config - if you try to build without `MBEDTLS_HAVE_ASM`, check_config won't complain but you will not get the AESCE implementation.

I guess we should remove the check for HAVE_ASM here (although technically we do still have the asm optimisation barrier for gcc)